inc/sainexthopgroup.h

```diff
@@ -59,22 +59,40 @@ typedef enum _sai_next_hop_group_attr_t
     /** Number of next hops in the group [uint32_t] */
     SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_COUNT,
 
+    /** Next hop list [sai_object_list_t] */
+    SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_MEMBER_LIST,
+
     /** READ-WRITE */
 
     /** Next hop group type [sai_next_hop_group_type_t] (MANDATORY_ON_CREATE|CREATE_ONLY) */
     SAI_NEXT_HOP_GROUP_ATTR_TYPE,
 
-    /** Next hop list [sai_object_list_t] (MAXDATORY_ON_CREATE) 
-     * The next hop group must have at least one next hop member at the creation time */
-    SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_LIST,
```

Comment:
We discussed about this but I am NOT convinced that leaving this as optional attribute creates a complicated behavior (in conjunction with a readonly member list retrieval attribute). 
But in the absence of the bulk create mechanism in 1.0 I do see this having two benefits: 
a. We don't break any existing test and application code. 
b. We have a simple and optimized way to create the nexthop group with a known list of nexthops up front instead of calling the new create member API multiple times.

Once the bulk create interface is defined post 1.0 this mechanism could be deprecated and the existing code could then be easily modified to use the new mechanism.